pkgs/tools/system/netdata/default.nix

```diff
@@ -59,6 +60,20 @@ in stdenv.mkDerivation rec {
     # Avoid build-only inputs in closure leaked by configure command:
     #   https://github.com/NixOS/nixpkgs/issues/175693#issuecomment-1143344162
     ./skip-CONFIGURE_COMMAND.patch
+
+    (fetchpatch {
+      # Fix stream authentication: https://github.com/netdata/netdata/pull/14063
+      name = "fix-stream-authentication";
+      url = "https://github.com/netdata/netdata/commit/811028aea2f146cc0ac2bc403f7d692add400d63.patch";
+      hash = "sha256-Evp5EXcBrdqNwigofequALxOvmUr62hCC5YtXcvBhao=";
+    })
+
+    (fetchpatch {
+      # Fix alarm notification commands: https://github.com/netdata/netdata/pull/14064
+      name = "fix-alarm-notification-commands";
```

Comment:
```suggestion
      # https://github.com/netdata/netdata/pull/14064
      name = "fix-alarm-notification-commands.patch";
```

Reply:
Found something for this one, what do you think?